README.md

```diff
@@ -143,3 +143,37 @@ search.addWidget(
 );
 ```
 
+### multipleChoiceList
+
+#### API
```

Comment:
I have faith we will be able to automatically parse it also to generate our API doc (not usage website/doc)